src/Network/Email/AbstractTransport.php

```diff
@@ -1,3 +1,16 @@
 <?php
-// @deprecated Backward compatibility with 2.x, 3.0.x
-class_alias('Cake\Mailer\AbstractTransport', 'Cake\Network\Email\AbstractTransport');
+// @codingStandardsIgnoreFile
+namespace Cake\Network\Email;
+
+class_alias(\Cake\Mailer\AbstractTransport::class, AbstractTransport::class);
+
+if (class_exists(AbstractTransport::class)) {
+    return;
+}
```

Comment:
I think you guys are overthinking this.

My changes are supposed to be non-breaking and non-executing code. It's only read by the IDE. When PHP executes it never reaches the class declaration before the `if(class_exists())`. The only way the PHP files would fail is if there was a syntax error, but there should be no run-time errors.